NebulaModel/DataStructures/PlayerData.cs

```diff
@@ -45,6 +48,20 @@ public void Serialize(INetDataWriter writer)
             Rotation.Serialize(writer);
             BodyRotation.Serialize(writer);
             Mecha.Serialize(writer);
+            writer.Put(Appearance != null);
+            if(Appearance != null)
+            {
+                using (MemoryStream ms = new MemoryStream())
+                {
+                    using (BinaryWriter wr = new BinaryWriter(ms))
+                    {
+                        Appearance.Export(wr);
+                    }
+                    byte[] export = ms.ToArray();
```

Comment:
Isn't `Appearance.ToByte()` a shortcut to this?

Reply:
It is, could probably be exchanged but would have the same effect.